drucker-prager documentation warning

I see the following warning that I don't know how to fix as I don't understand the argument to the function:
```
/home/bob/source/include/aspect/material_model/rheology/drucker_prager.h:72: warning: argument 'n_fields' of command @param is not found in the argument list of aspect::MaterialModel::Rheology::DruckerPrager< dim >::parse_parameters(ParameterHandler &prm, const std::unique_ptr< std::vector< unsigned int >> &expected_n_phases_per_composition=nullptr)
/home/bob/source/include/aspect/material_model/rheology/drucker_prager.h:83: warning: The following parameters of aspect::MaterialModel::Rheology::DruckerPrager::parse_parameters(ParameterHandler &prm, const std::unique_ptr< std::vector< unsigned int >> &expected_n_phases_per_composition=nullptr) are not documented:
  parameter 'expected_n_phases_per_composition'
```
See
https://github.com/geodynamics/aspect/blob/d73947ccb7de1cf1dc87aeca8981f90ae2235cd7/include/aspect/material_model/rheology/drucker_prager.h#L76-L84

--- a/include/aspect/material_model/rheology/drucker_prager.h
+++ b/include/aspect/material_model/rheology/drucker_prager.h
@@ -69,13 +69,11 @@ namespace aspect
           declare_parameters (ParameterHandler &prm);
 
           /**
-           * Read the parameters from the parameter file. These parameters might
-           * be modified outside of this class for each call to the functions below,
-           * so the read parameters are returned to the caller instead of stored as members.
-           *
-           * @param [in] n_fields The number of expected values for the angle of friction and
-           * cohesion lists. Generally either the number of compositional fields or this number
-           * plus one (for a background field), depending on how the user handles background fields.
+           * Read the parameters this class declares from the parameter file.
+           * If @p expected_n_phases_per_composition points to a vector of
+           * unsigned integers, this is considered the number of phases
+           * for each compositional field (plus possibly a background field)
+           * and this number will be checked against the parsed parameters.
            *
            * @param [in] prm The ParameterHandler to read from.
            */

